tests/test_content.py

```diff
@@ -260,6 +260,28 @@ def test_number_padding_3():
     assert old_result == '0042'  # output
 
 
+@pytest.mark.skipif(sys.version_info < (2, 7), reason="Requires Python 2.7")
+def test_number_separator():
+    """
+    String formatting allows thousands separators using commas.
+    This works for floating point and integer numbers.
+    """
+    new_result = '{:,}'.format(42042.042)
+
+    assert new_result == '42,042.042'
+
+
+@pytest.mark.skipif(sys.version_info < (3, 6), reason="Requires Python 3.6")
```

Comment:
Since these changes more recent version of Python, I added a `pytest.mark` to skip the unsupported versions. If you would prefer a different way of conditionally running tests, let me know.

Reply:
I had no luck running `main.py` locally. I wanted to see the resulting html for myself. To the best of my predictions this will look correct on the generated website.
> `AttributeError: module '_ast' has no attribute 'Str'`